.circleci/config.yml

```diff
@@ -0,0 +1,55 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.6.3-node-browsers
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "rubyhub.gemspec" }}
+            - v1-dependencies-
+
+      - run:
+          name: install dependencies
+          command: |
+            gem update --system
+            gem install bundler
+            bundle install --jobs=4 --retry=3 --path vendor/bundle
+
+      - run:
+          name: Rubocop
+          command: bundle exec rubocop
+
+      - run:
+          name: Fasterer
+          command: bundle exec fasterer
+
```

Comment:
I'd suggest to move it to another [workflow](https://github.com/rubygarage/circledge/blob/master/intro_to_circleci.md#workflows) 'linters'.